magefile.go

```diff
@@ -25,7 +26,19 @@ func Swag() error {
 
 // Test runs the tests.
 func Test() error {
-	return commands.Test("./...")
+	res, err := sh.Output("go", "list", "./...")
+	if err != nil {
+		return err
+	}
+	splits := strings.Split(res, "\n")
+	args := []string{"test", "-race", "-cover"}
+	for i := range splits {
+		if strings.Contains(splits[i], "e2e") {
+			continue
+		}
+		args = append(args, splits[i])
+	}
+	return sh.RunV("go", args...)
```

Comment:
Honestly you could just run: `sh.RunV("go", "test", "-race", "-cover", "-short", "./...")`
And then in E2E ignore if flag `-short` is present.

Here: https://github.com/mysteriumnetwork/pilvytis/blob/master/e2e/pilvytis/common_test.go#L17

Seems a bit cleaner 🤷🏻